### DISCUSSION TITLE
Editor Team Question: Multi(creator) Experience

### DISCUSSION THREAD

**Discussion by minecraftchloe** [Mojang/minecraft-editor#5] [Q&A]
What feels right? What feels wrong?

**Comment by ihategravel2** [comment]
Overall, it feels right. I assume the camera is moving at the same speed as a flying player. However, I think the camera needs to move significantly faster. The only thing that doesn't feel right is that you can't use WASD without right-clicking your mouse.

**Reply by SineVector241** [reply]
Would be cool if instead of just setting a static/constant movement you can modify how fast your moving either by setting it in some sort of ui setting or by a keybind.
